### DISCUSSION TITLE
Do we have class during midterm and final week?

### DISCUSSION THREAD

**Discussion by iJinjin** [seominjoon/kaist-ai605#31] [Q&A] [answered]
Hello professor,

I was taking a look at the course schedule and found that we have lectures planned during midterm (4/19 ~ 4/23) and final week (6/7 ~ 6/11). During the exam week, courses with exams have pre-assigned time slots which are usually different from the regular class times, and all classes in KAIST do not give lectures during exam week with very few exceptions. I think that many students taking this course (including myself) may have schedule conflicts with their own exams. It would be grateful if you could give us some more information on this course schedule. Thank you in advance!

**Comment by seominjoon** [comment, marked as answer]
I also just realized this (my apology, it's my first semester at KAIST). I am canceling both classes next week, but I will upload a few list of papers that you need to read before you come to class on Monday April 26th.  I will also make an announcement on KLMS. Thanks for letting me know.

**Reply by iJinjin** [reply]
Thank you professor!
